transports/webrtc-utils/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+edition = "2021"
+name = "libp2p-webrtc-utils"
+rust-version.workspace = true
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bytes = "1"
+futures = "0.3"
+hex = "0.4"
+js-sys = { version = "0.3", optional = true }
+libp2p-core = { workspace = true }
+libp2p-identity = { workspace = true }
+libp2p-noise = { workspace = true }
+log = "0.4.19"
+quick-protobuf = "0.8"
+quick-protobuf-codec = { workspace = true }
+serde = { version = "1.0", features = ["derive"] }
+sha2 = "0.10.7"
+thiserror = "1"
+tinytemplate = "1.2"
+wasm-bindgen = { version = "0.2.87", optional = true }
+web-sys = { version = "0.3.64", optional = true, features = [
+    "RtcSdpType",
+    "RtcSessionDescription",
+    "RtcSessionDescriptionInit",
+] }
+webrtc = { version = "0.8.0", optional = true }
+
+[features]
+tokio = ["dep:webrtc"]
```

Comment:
I agree, but I'm not sure if that's possible without duplicating the code from `webrtc-rs`. We'd have to copy over the SDP code from `webrtc-rs` into `libp2p-webrtc-utils`. Is that something we're willing to maintain?

Reply:
and `webrtc::dtls_transport::dtls_fingerprint::RTCDtlsFingerprint` too

Reply:
We need to create a `Fingerprint` wrapper type in `libp2p-webrtc` anyway because we cannot just export `libp2p_webrtc_utils::Fingerprint` as that would leak it into the public API and any breaking change to `libp2p-webrtc-utils` would be a breaking change to `libp2p-webrtc`.

Thus, if we already have a utils fingerprint type and the higher-level one, we can move the constructor function that is currently feature-gated to the `libp2p-webrtc` one, do all of the parsing from `RTCDtlsFingerprint` there and construct the internal `libp2p_webrtc_utils::Fingerprint`. Does that make sense?

Reply:
Fixed.